### PR TITLE
Move user-manuals to install-manuals and add real user-manuals

### PR DESCRIPTION
**Pull request by robojones**
Part of Gamify-IT/issues#309

### Contents of this PR
- Move the contents of `user-manuals/` to `ops-manuals/`
- Create an actual user manual (meant for lecturers or students) for chickenshock

### Reasoning
What we currently have in the `user-manuals/` folder is only of concern for people hosting our software (ops / operations). Therefore I renamed the directory `user-manuals/` to `ops-manuals/`.

In our code review we got the feedback, that we are missing high-level, non-technical documentation for people unrelated to the project. I want to repurpose the `user-manuals/` directory to collect documentation for our **actual users**, meaning **lecturers and students**. The documentation should describe, how the overworld, minigames and lecturer interface work (how to play the games and how to use the UI). It should contain screenshots and useful diagrams. We can then reference this documentation from the minigame repository README.md files and other resources.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # Docs
 
-This repo contains all documentation for the minigame platform. Repos might contain further information but all information in this repo should be kept up to date.
+This repo contains all documentation for the minigame platform. Repos might contain further information but all
+information in this repo should be kept up to date.
```
[the remark this answers is not in the document]
Comment (robojones) [reply README.md:4]: I don't like it when the lines get too long

File: index.rst [hunk 2]
```diff
@@ -1,7 +1,7 @@
 .. Gamify IT documentation master file, created by
-   sphinx-quickstart on Thu May 26 13:13:28 2022.
-   You can adapt this file completely to your liking, but it should at least
-   contain the root `toctree` directive.
+sphinx-quickstart on Thu May 26 13:13:28 2022.
+You can adapt this file completely to your liking, but it should at least
+contain the root `toctree` directive.
```
Comment (delvh) [inline index.rst:4]: Probably not a good idea.
I think you just uncommented this block with this action, but I'm also no expert on RST.

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (robojones) [reply README.md:4]: It's IntelliJ auto-formatting

[the remark this answers is not in the document]
Comment (delvh) [reply README.md:4]: The problem is, depending on the Markdown reader, this arbitrary line break will also present in the rendered markdown…

**Comment by delvh** [comment]
Please move `ops-manuals/maps` back to `user-manuals/maps`. They are needed there for the lecturer interface, and they belong in `user-manuals`.

**Comment by delvh** [comment]
Oh, also:
I will push a commit in the next minutes to purge the `png`s out of this repo before they ever get merged.

**Review by LeviUniAcc — APPROVED** [review]
i like especially the chickenshock user-manuals

File: user-manuals/minigames/chickenshock.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+Each chicken has a different answer to the question floating above them.
+The player has to shoot at the chicken with the correct answer.
+After the player chooses an answer, the next question is presented.
+
+![end screen screenshot](assets/chickenshock_end_screen.webp)
+With each question, the player can earn points.
+The goal is to get as many points as possible.
+The game ends when the player has answered all questions or if the timer in the top left corner runs out.
+Then, the player is presented with the score screen.
+
```
Comment (LeviUniAcc) [inline user-manuals/minigames/chickenshock.md:24]:
```suggestion

Pressing `p` returns the mouse.

```

**Review by Gr33ndev — APPROVED** [review]

File: install-manuals/README.md [hunk 4]
```diff
@@ -0,0 +1,4 @@
+# User manuals
```
Comment (delvh) [inline install-manuals/README.md:1]: Not anymore.

File: user-manuals/README.md [hunk 5]
```diff
@@ -1,3 +1,12 @@
 # User manuals
 
-Contains instructions on how a user can start the specific service.
+User manuals contains descriptions and screenshots of the different components (overworld, minigames, lectuer interface)
```
Comment (delvh) [inline user-manuals/README.md:3]:
```suggestion
The user manuals contain descriptions, screenshots, and tutorials on how to use the different components (overworld, minigames, lecturer interface) 
```

File: user-manuals/minigames/chickenshock.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+The player has to shoot at the chicken with the correct answer.
+After the player chooses an answer, the next question is presented.
+
+![end screen screenshot](assets/chickenshock-end-screen.webp)
+With each question, the player can earn points.
+The goal is to get as many points as possible.
+The game ends when the player has answered all questions or if the timer in the top left corner runs out.
+Then, the player is presented with the score screen.
+
+Pressing `p` returns the mouse.
```
Comment (delvh) [inline user-manuals/minigames/chickenshock.md:25]:
```suggestion
Pressing <kbd>p</kbd> unlocks the mouse.
```
I'm quite confused by this sentence. Is it correct like that?

File: install-manuals/README.md (hunk 4 above)
Comment (robojones) [inline install-manuals/README.md:1]:
```suggestion
# Install manuals
```

**Review by cdingm — APPROVED** [review]

**Review by delvh — APPROVED** [review]

**Pull request merged**